manifests/charts/istiod-remote/files/injection-template.yaml

```diff
@@ -252,6 +252,10 @@ template: |
     - name: {{ $key }}
       value: "{{ $value }}"
     {{- end }}
+    {{- range $key, $val := .Values.global.proxy.env }}
```

Comment:
This is already done by `.ProxyConfig.ProxyMetadata` just 4 lines above. Using this option is preferred because it already exists as part of a proper versioned API and can be overriden with annotations

Reply:
+1 ( left similar comments before seeing this )